packages/11ty/content/_assets/styles/components/q-lightbox-slides.scss

```diff
@@ -44,6 +44,36 @@
       justify-content: center;
     }
 
+    &--table {
+
```

Comment:
```suggestion
      .overflow-container {
        background-color: white;
        color: black;
        display: flex;
        flex-direction: column;
        max-height: 80vh;
        max-width: 80vw;
        padding: 1rem;
      }
```

Reply:
I would still keep the `.overflow-container` overrides nested under `&--table`, mostly because it applies background color/dimensions/text color that may only be applicable to table content (I could potentially see a different treatment for non-table overflow containers) but other than this extremely picky nit, I think this all looks good to go

Reply:
@anderspollack thanks. I made that change. Good to merge in now?